src/pages/signup/StepThree.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import Button from '@mui/material/Button';
+
+
+export default function StepThree() {
+  return (
+    <div className="StepThreeForm">
+        <FormControl sx={{width: 400}} className="StepThreeInput">
+            <InputLabel>Gender</InputLabel>
+            <Select
+                labelId="demo-simple-select-label"
+                id="demo-simple-select"
+                value={"gender"}
+                label="Gender"
+                // onChange={handleChange}
+            >
+                <MenuItem value={"male"}>Male</MenuItem>
+                <MenuItem value={"female"}>Female</MenuItem>
+                <MenuItem value={"other"}>Other</MenuItem>
+            </Select>
+        </FormControl>
+
+        <FormControl sx={{width: 400}}>
+        <InputLabel>Select occupation</InputLabel>
+            <Select
+                labelId="demo-simple-select-label"
+                id="demo-simple-select"
```

Comment:
For all of these select dropdowns, we should add a `name` prop that matches the `id`
```suggestion
                id="applicant-occupation"
                name="applicant-occupation"
```